app/services/make_an_offer.rb

```diff
@@ -28,6 +31,8 @@ def save
 
 private
 
+  attr_reader :application_choice
```

Comment:
Is this needed?

Reply:
We were still using instance variable,  I'm gonna change `@application_choice` -> `application_choice`